pynsee/geodata/get_geodata.py

```diff
@@ -25,6 +26,8 @@ def get_geodata(
 
         constrain_area (:class:`~geopandas.GeoDataFrame`, optional): GeoDataFrame used to constrain the area of interest. Defaults to None.
 
+        silence (bool, optional): whether to print warnings or not. Defaults to False.
+
```

Comment:
could you add a ``.. versionadded:: 0.3.0`` with "added `silent` parameter"?

Reply:
Sure, sorry I have not the reflex to log docstrings (yet). Are we certain we have enough changes to justify a jump to 0.3 version (this seems very minor to me).

Reply:
I just assumed we would not necessarily tag a new version until we have enough changes, so this would become 0.3.0...

Reply:
Noted. Though the geodata PR from this morning might be enough to justify a new version. I still don't know if this is a frequent behaviour, but that was troubling enough...